IS0801Test.py

```diff
@@ -12,12 +12,405 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from GenericTest import GenericTest
+from GenericTest import GenericTest, NMOSTestException
+from TestHelper import compare_json
+from TestResult import Test
+from NMOSUtils import NMOSUtils
+from is08 import Action, Activation
+from is08.inputs import getInputList
+from is08.outputs import getOutputList
+from is08.active import Active
+from is08.io import IO
+from is08.testConfig import globalConfig
+import time
+import copy
+
+MAPPING_API_KEY = "channelmapping"
 
 
 class IS0801Test(GenericTest):
     """
     Runs IS-08-01-Test
     """
+
     def __init__(self, apis):
         GenericTest.__init__(self, apis)
+        globalConfig.apiUrl = apis[MAPPING_API_KEY]['url']

```

Comment:
General comment (don't change it here), but as it may be of interest, PEP8 suggests snake case for function/var names: https://www.python.org/dev/peps/pep-0008/#function-and-variable-names

Reply:
Huh, how odd. Fair enough! So used to doing it this way on our code to match style I didn't realise.